WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Method/SimplePaymentsMethod.swift

```diff
@@ -40,23 +40,23 @@ struct SimplePaymentsMethod: View {
             Divider()
 
             Group {
-                MethodRow(icon: .priceImage, title: Localization.cash) {
+                MethodRow(icon: .priceImage, title: Localization.cash, showBetaAnnotation: false) {
                     showingCashAlert = true
                     viewModel.trackCollectByCash()
                 }
 
                 if viewModel.showPayWithCardRow {
                     Divider()
 
-                    MethodRow(icon: .creditCardImage, title: Localization.card) {
+                    MethodRow(icon: .creditCardImage, title: Localization.card, showBetaAnnotation: false) {
```

Comment:
Is there no such thing as default values in Swift UI? 😅

Reply:
They do exist lol, but as this change is temporary I didn't bother with adding a custom initializer, this one is auto synthesized!